### PR TITLE
Adding Elasticsearch demo

### PR DESCRIPTION
**Pull request by rusher81572**
Adding the Elasticsearch demo for the blog post.

### CODE REVIEW

File: elasticsearch-scylla/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+Three Scylla 1.7 nodes
+
+##### Elasticsearch Architecture
+Two Elasticsearch nodes (Master, data).
+![Pic](https://raw.githubusercontent.com/scylla/scylla-code-samples/master/elasticsearch-scylla/diagram.png)
+
+### Prerequisites
+
+1. Docker for [Mac](https://download.docker.com/mac/stable/Docker.dmg) or [Windows](https://download.docker.com/win/stable/InstallDocker.msi).
+2. This Git [Repo](https://github.com/rusher81572/elasticsearch-scylla/archive/master.zip)
```
Comment (tzach) [inline elasticsearch-scylla/README.md]: @rusher81572 wouldn't it be easier to clone this repo instead of downloading and unziping a file?

**Comment by rusher81572** [comment]
@tzach Changed README to say "2. This Git repository" instead.

**Comment by tzach** [comment]
> @tzach Changed README to say "2. This Git repository" instead. 

If I'm using the git repo, do I need to run `unzip elasticsearch-scylla-master.zip` in the next step?

File: elasticsearch-scylla/README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+##### Scylla Architecture
+Three Scylla 1.7 nodes
+
+##### Elasticsearch Architecture
+Two Elasticsearch nodes (Master, data).
+![Pic](https://raw.githubusercontent.com/scylla/scylla-code-samples/master/elasticsearch-scylla/diagram.png)
+
+### Prerequisites
+
+1. Docker for [Mac](https://download.docker.com/mac/stable/Docker.dmg) or [Windows](https://download.docker.com/win/stable/InstallDocker.msi).
```
Comment (tzach) [inline elasticsearch-scylla/README.md]: 1. many of our users are running Linux. They can find the install themselves
2. You are using need Install Docker Compose as well.

File: elasticsearch-scylla/README.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+4. (Optional) [Twitter API credentials](https://dev.twitter.com/)
+
+### Building the images
+```
+unzip elasticsearch-scylla-master.zip
+cd elasticsearch-scylla
+docker-compose build
+```
+
+To use the Twitter app to mine data from Twitter, modify the twitter section of docker-compose.yml with your developer API credentials and desired Twitter topic.
```
Comment (tzach) [inline elasticsearch-scylla/README.md]: how do I do that?
Is it on this page: https://apps.twitter.com/app/new
what values should I use?

File: elasticsearch-scylla/README.md (hunk 2 above)
Comment (rusher81572) [inline elasticsearch-scylla/README.md]: Updated README

File: elasticsearch-scylla/README.md (hunk 3 above)
Comment (rusher81572) [inline elasticsearch-scylla/README.md]: Updated README

File: elasticsearch-scylla/README.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+0d7f66c4b1c9        elasticsearchscylla_elasticsearch-slave1   "/bin/sh -c 'bash ..."   49 seconds ago      Up 46 seconds                                elasticsearchscylla_elasticsearch-slave1_1
+b5b04ac809b8        elasticsearchscylla_elasticsearch-master   "/bin/sh -c 'bash ..."   49 seconds ago      Up 45 seconds       0.0.0.0:9200->9200/tcp   elasticsearchscylla_elasticsearch-master_1
+f4e03f46a8ea        elasticsearchscylla_scylla-node2           "/bin/sh -c 'bash ..."   49 seconds ago      Up 47 seconds                                elasticsearchscylla_scylla-node2_1
+d2bfecf52f4d        elasticsearchscylla_scylla-node1           "/bin/sh -c 'bash ..."   49 seconds ago      Up 46 seconds                                elasticsearchscylla_scylla-node1_1
+```
+
+### Checking the Twitter data from Scylla with cqlsh
+```
+docker exec -it elasticsearchscylla_scylla-node1_1 cqlsh
+cqlsh> select * from twitter.tweets;
```
Comment (tzach) [inline elasticsearch-scylla/README.md]: I suggest using 
```select * from twitter.tweets LIMIT 10;```
or similar. Query for the entire data may timeout

File: elasticsearch-scylla/README.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+```
+
+### To start writing the tweets to Scylla again
+```
+curl http://127.0.0.1:8080/start
+```
+
+### Accessing Kibana to view the Twitter data
+1. Goto https://0.0.0.0:5601 in your web browser
+2. Enter logstash for the index name and then click the create button
```
Comment (tzach) [inline elasticsearch-scylla/README.md]: I do not see logstash in the dashboard. This is what I get
image

All docker instances are up and running
```
$ sudo docker ps
CONTAINER ID        IMAGE                                      COMMAND                  CREATED             STATUS                           PORTS                    NAMES
81526c2c6a62        elasticsearchscylla_twitter                "/bin/sh -c 'node ..."   9 minutes ago       Up 8 minutes                     0.0.0.0:8080->8080/tcp   elasticsearchscylla_twitter_1
b249ac3af2cc        elasticsearchscylla_kibana                 "/bin/sh -c 'cd /k..."   9 minutes ago       Up 8 minutes                     0.0.0.0:5601->5601/tcp   elasticsearchscylla_kibana_1
2a0fb4a55e43        elasticsearchscylla_scylla-node1           "/bin/sh -c 'bash ..."   9 minutes ago       Up 8 minutes                                              elasticsearchscylla_scylla-node1_1
5f5ecdbd59a9        elasticsearchscylla_elasticsearch-master   "/bin/sh -c 'bash ..."   9 minutes ago       Up 1 second                      0.0.0.0:9200->9200/tcp   elasticsearchscylla_elasticsearch-master_1
cbee1fc66495        elasticsearchscylla_scylla-node3           "/bin/sh -c 'bash ..."   9 minutes ago       Up 8 minutes                                              elasticsearchscylla_scylla-node3_1
8f3103e9f1cb        elasticsearchscylla_elasticsearch-slave1   "/bin/sh -c 'bash ..."   9 minutes ago       Restarting (78) 59 seconds ago                            elasticsearchscylla_elasticsearch-slave1_1
f72df4b33ffc        elasticsearchscylla_scylla-node2           "/bin/sh -c 'bash ..."   9 minutes ago       Up 8 minutes                                              elasticsearchscylla_scylla-node2_1
```

File: elasticsearch-scylla/README.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+##### Elasticsearch Architecture
+Two Elasticsearch nodes (Master, data).
+![Pic](https://raw.githubusercontent.com/scylla/scylla-code-samples/master/elasticsearch-scylla/diagram.png)
+
+### Prerequisites
+
+1. Docker for [Mac](https://download.docker.com/mac/stable/Docker.dmg) or [Windows](https://download.docker.com/win/stable/InstallDocker.msi).
+2. This Git repository
+3. 3GB of RAM or greater for Docker
+4. Twitter API credentials. After signing up for a developer [account](https://dev.twitter.com/), go to the Twitter Apps [page](https://apps.twitter.com/) and click on “Create New App”. The API credentials are provided after the application is created.
```
Comment (tzach) [inline elasticsearch-scylla/README.md]: Is the signing up for a developer [account](https://dev.twitter.com/) a must?
I only created a new apps and it seems to work

Comment (rusher81572) [inline elasticsearch-scylla/README.md]: According to all the instructions that I found online, yes.

File: elasticsearch-scylla/README.md (hunk 5 above)
Comment (rusher81572) [inline elasticsearch-scylla/README.md]: Depending on the computer that you run this demo on, it make take a few minutes for elasticsearch to get up and running. Did it ever come up?

File: elasticsearch-scylla/README.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+curl http://127.0.0.1:8080/stop
+```
+
+### To start writing the tweets to Scylla again
+```
+curl http://127.0.0.1:8080/start
+```
+
+### Accessing Kibana to view the Twitter data
+1. Goto https://0.0.0.0:5601 in your web browser
```
Comment (tzach) [inline elasticsearch-scylla/README.md]: `https://0.0.0.0:5601` did not work for me, only `http://0.0.0.0:5601` (http not http**s** )

**Comment by tzach** [comment]
My problem was in ES logs:
```
Max virtual memory areas vm.max_map_count [65530] likely too low, increase to at least [262144]
```
and the solution was
```
sudo sysctl -w vm.max_map_count=262144
```

But now I have a new issue
kibana fail to connect to ES
image

ES logs seems fine
On Kibana logs I see 404 response from ES
```
"remoteAddress":"127.0.0.1","userAgent":"127.0.0.1","referer":"http://localhost:5601/eqo/app/kibana"},"res":{"statusCode":404,"responseTime":12,"contentLength":9},"message":"GET /api/index_patterns/_fields_for_wildcard?pattern=logstash-*&meta_fields=_source&meta_fields=_id&meta_fields=_type&meta_fields=_index&meta_fields=_score 404 12ms - 9.0B"}
```

@rusher81572 any idea?

**Comment by rusher81572** [comment]
According to the instructions, the pattern should be logstash, not logstash-*

File: elasticsearch-scylla/README.md (hunk 7 above)
Comment (rusher81572) [inline elasticsearch-scylla/README.md]: Kibana may take a few minutes to get up and running after starting the container. https://0.0.0.0:5601 is the only way that I connect to it successfully.

**Comment by tzach** [comment]
Thanks @rusher81572 
I made a minor change to README https://github.com/scylladb/scylla-code-samples/pull/7/commits/60bf0ecbf3b2f993aca4405d24ebf7e3343ccd56 before merging
It did not work for me without this change

**Pull request merged**